Javascript/QuizApp/features.js

```diff
@@ -1,6 +1,31 @@
 const formInput = document.querySelector(".question-input-form");
 const answersContainer = document.querySelector(".answers-container");
 const buttonDiv = document.querySelector(".btn-wrapper");
+let quizQuestionArray = [];
+
+// I created a global variable to store the quiz questions array after fetching it from the API.
+// I did this so that I don't need to change the quiz questions array reference in various functions.
+// Code is working smooth.
+// My app still save the new questions in the quizQuestionArray and display them in the list of questions (Not Saving to API just in memory).
+// So my app is not fully functional (it saves new questions only during run time in quizQuestionArray variable which already has fetched data).
+// I will work on it in future to save the new questions in API file with Method POST after learning all methods related to APIs.
```

Comment:
> It saves new questions only during run time 

It is what we expect from your projects. The goal is to work with javascript in the browser, not set up an infrastructure with a database, an API hosted on a server, etc.

> I will work on it in future to save the new questions in API file with Method POST after learning all methods related to APIs.

You will probably hear about this during the Node.js module, but it will not be part of this project assignment. You are using a remote JSON file to simulate fetching data from an API. It can not handle POST requests. But feel free to come back to this project in the future and add a proper API to it 😃